LogMonitor/src/LogMonitor/LogFileMonitor.h

```diff
@@ -43,26 +47,23 @@ enum class EventAction
     Unknown = 5,
 };
 
-
 struct DirChangeNotificationEvent
 {
     std::wstring FileName;
     EventAction Action;
     UINT64 Timestamp;
 };
 
-
 class LogFileMonitor final
 {
 public:
     LogFileMonitor() = delete;
 
     LogFileMonitor(
-        _In_ const std::wstring& LogDirectory,
-        _In_ const std::wstring& Filter,
+        _In_ const std::wstring &LogDirectory,

```

Comment:
nitpick: I think the `&` should right after the type as per common convention; `T&`?